src/tools/miri/src/bin/miri.rs

```diff
@@ -309,6 +308,9 @@ fn main() {
         )
     }
 
+    // Add an ICE bug report hook.
+    rustc_driver::install_ice_hook(Some("https://github.com/rust-lang/miri/issues/new"), None);
```

Comment:
Do we usually link to the 'new issue' page or just the issue list? IMO the latter makes more sense to avoid duplicates.

Reply:
For rustc we use https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md, so I'd say we usually link 'new issue'.

I guess the point is that duplicates are better than an unreported problem.